app/src/main/java/de/westnordost/streetcomplete/quests/collection_times/AddCollectionTimesAdapter.java

```diff
@@ -0,0 +1,396 @@
+package de.westnordost.streetcomplete.quests.collection_times;
+
+import android.content.Context;
+import android.content.DialogInterface;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.widget.RecyclerView;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.text.DateFormatSymbols;
+import java.util.ArrayList;
+import java.util.List;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.meta.CountryInfo;
+import de.westnordost.streetcomplete.view.dialogs.AlertDialogBuilder;
+import de.westnordost.streetcomplete.view.dialogs.RangePickerDialog;
+
+public class AddCollectionTimesAdapter extends RecyclerView.Adapter
```

Comment:
Is this whole file really needed? Can't the one from the opening_hours quest be reused?

Reply:
No, the opening hours one shows ranges, while collection times are ... times.